src/celutil/utf8.cpp

```diff
@@ -772,39 +772,45 @@ std::string ReplaceGreekLetterAbbr(const std::string& str)
     Greek *instance = Greek::getInstance();
     std::string ret = str;
 
-    if (str[0] >= 'A' && str[0] <= 'Z' &&
-        str[1] >= 'A' && str[1] <= 'Z')
+    if (str[0] >= 'A' && str[0] <= 'Z')
```

Comment:
Could you explain what's wrong with this file?

Reply:
I added replacement also for Greek letters names, not only for cannonical abbreviations.

Reply:
Ok. Could you move this change into its own commit with the small explanation as above?